### DISCUSSION TITLE
Add UserScript feature

### DISCUSSION THREAD

**Discussion by SkeStars** [plateaukao/einkbro#277] [Ideas]
Are there any plans to add a userscripts feature to this awesome browser? This would allow me to easily customize the CSS style for each website, to remove annoying ads and recommendations. The implementation would be similar to Via browser and X browser. ( In addition to userscripts, those two browsers also have the function of long-pressing on interface elements to identify and block ads.)

**Comment by Spiralo-Idioide** [comment]
I also think it would be a really great feature. 

It would allow every user to add a lot of the features they need by themselves instead of making feature requests ! This might help make the browser a lot more powerful without adding  too much burden to maintain.

**Comment by plateaukao** [comment]
supported
